hyperdash/constants.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+AUTH_HEADER_KEY = "x-hyperdash-auth"
+WAMP_ENDPOINT = "/api/v1/sdk/wamp"
+WAMP_REALM = u"hyperdash.v1.sdk"
+
+
+def get_base_url():
+    environment = os.environ.get("HYPERDASH_ENVIRONMENT")
+    if environment == "dev":
+        return u"ws://127.0.0.1:4000"
+    return u"wss://hyperdash.io"
```

Comment:
Are we hard coding the URL here? Why not use an env for the URL (e.g., `HYPERDASH_SERVER`).